plasmapy/physics/distribution.py

```diff
@@ -60,6 +65,8 @@ def Maxwellian_1D(v: units.m/units.s,
 
     .. math::
     f = \sqrt{\frac{m}{2 \pi k_B T}} \exp(-\frac{m}{2 k_B T} (v-V)^2)
+    f = (\pi * v_Th^2)^{-1/2} \exp(-(v - v_{drift})^2 / v_Th^2)
```

Comment:
Could we add something like "or equivalently," or "two equivalent forms:" here?

That might break with the `.. math::` .rst tag. We'll have to keep an eye on that.

Reply:
Oh, I was intending to remove the first form listed there. I think it makes more sense to look at the function as a comparison of velocities to obtain a probability with respect to the most probably velocity rather than having temperatures, and masses floating around. We can alway add the thermal velocity function under "see also"

Reply:
It probably wouldn't hurt too much to have both of those available, but it's not critical to have both here, I suppose!